www/src/public/js/controllers/builder/main.js

```diff
@@ -313,11 +319,14 @@
                 baseStats.amulet = encoder.toNumber(listStr.slice(0,1));
             listStr = listStr.substring(1);
 
-            if (statList[0] === '_')
-                baseStats.hazelnut = -1;
-            else
-                baseStats.hazelnut = encoder.toNumber(listStr.slice(0,1));
-            listStr = listStr.substring(1);
+            // v3: Hazelnut
+            if (listVersion == 3) {
+                if (statList[0] === '_')
+                    baseStats.hazelnut = -1;
+                else
+                    baseStats.hazelnut = encoder.toNumber(listStr.slice(0,1));
+                    listStr = listStr.substring(1);
```

Comment:
be careful of the indenting here, while it doesn't pose an issue, it might be confusing to another reader. They might think that the line was intended to be inside of the else statement, which might lead them to putting brackets around lines 327-328 and that would break things.

Reply:
Duh moment on my part but good eye.  Fixed.